js/components/generate/GenerateForm.js

```diff
@@ -296,91 +298,116 @@ export default class GenerateForm extends React.Component {
-                  component="div"
-                />
-
-                <label htmlFor="taken">Do not show these courses</label>
-                <Field
-                  id="taken"
-                  name="taken"
-                  type="text"
-                  placeholder="E.g., CSC207H1, CSC236H1"
-                  style={{ marginBottom: errors.taken ? "0" : "1em" }}
-                />
-                <ErrorMessage className="error-message" name="taken" component="div" />
-
-                <label htmlFor="maxDepth">
-                  Depth of prerequisite chain (0 shows all prerequisites)
-                </label>
-                <p>
+                <div className="form-section">
+                  <h1 id="header-title" className="section-title">Search Course(s)</h1>
+
                   <Field
-                    id="maxDepth"
-                    name="maxDepth"
-                    type="number"
-                    min="0"
-                    step="1"
-                    style={{ marginBottom: "1em" }}
-                  />
-                </p>
-
-                {/* <label htmlFor="location">Campus</label>
-                <Field id="location" name="location" as="select" multiple
-                  style={{ verticalAlign: 'text-top', marginLeft: '1em', marginBottom: '1em', color: 'black' }}>
-                  <option value="utsg">St. George</option>
-                  <option value="utm">Mississauga</option>
-                  <option value="utsc">Scarborough</option>
-                </Field>
-
-                <p>
-                  <label htmlFor="includeRaws">Include non-course prerequisites</label>
-                  <Field id="includeRaws" name="includeRaws" type="checkbox"
-                    style={{ marginLeft: '1em', verticalAlign: 'middle' }}
+                    id="courses"
+                    name="courses"
+                    type="text"
+                    placeholder="e.g., CSC207H1, CSC324H1"
                   />
-                </p>
+                  <div className="error-container">
+                    <ErrorMessage
+                      className="error-message"
+                      name="courses"
+                      component="div"
+                    />
+                  </div>
+                </div>
 
-                <label htmlFor="includeGrades">Include grade-based prerequisites</label>
-                <Field id="includeGrades" name="includeGrades" type="checkbox"
-                  style={{ 'margin-left': '1em', 'vertical-align': 'middle' }} /> */}
+                <div className="form-section">
+                  <h2 id="filter-title" className="section-title">Filters</h2>
+
+                  <div className="title-container">
+                    <label htmlFor="departments">Departments</label>
+                    <a
+                      data-tooltip-id="departments-tooltip"
+                      data-tooltip-content="Only include courses from these departments"
```

Comment:
Add an extra sentence to say that each department code must be 3 letters. (I.e., we're telling the user the correct format here rather than in the error message.)

Similarly, add a help message for the "Search Course(s)" label saying that each code must be in the form CSC108H1 (or something similar).